src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitlabInfoContributor.java

```diff
@@ -12,7 +14,7 @@
 import de.tum.in.www1.artemis.config.Constants;
 
 @Component
-@Profile("gitlab")
+@Profile(PROFILE_GITLAB)
```

Comment:
How is this GitLab stuff related to the feature of this Pull Request?
In general, introducing constant here is not bad, but it seems out of scope here to me.